src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -117,4 +117,14 @@ public DirectoryController(DirectoryService service) {
     public ResponseEntity<Flux<ElementAttributes>> getElementsAttributes(@RequestParam("id") List<UUID> ids) {
         return ResponseEntity.ok().body(service.getElementsAttribute(ids));
     }
+
+    @PutMapping(value = "/directories/{elementUuid}/updateTypeNotification")
+    @Operation(summary = "send update type notification")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "The notification has been sent"),
+    })
+    public ResponseEntity<Mono<Void>> sendUpdateTypeNotification(@PathVariable("elementUuid") UUID elementUuid,
```

Comment:
We should rename this I think, see next comment.

Reply:
done